src/components/NoteEditor/NoteEditor.jsx

```diff
@@ -1,102 +1,147 @@
-import { useState } from "react";
+import { useNote } from "../../contexts";
 import { IcRoundAddCircleOutline } from "../../data/Icon";
 import { bxIcons } from "../../data/icons";
 import { ResizableTextArea } from "../UI/Input/ResizableTextArea";
 import "./NoteEditor.css";
+import { useState } from "react";
+import { Alert } from "../UI/Alert/Alert";
+import { useAxios } from "../../customHooks";
+
+export const NoteEditor = () => {
+	const { noteState, noteDispatch, inititalNoteObj, palette, addToNoteList } =
+		useNote();
+	const { noteObj } = noteState;
+	const { noteColor, dateCreated } = noteObj;
+
+	const { axiosRequest } = useAxios();
+	const [isPaletteVisible, setIsPaletteVisible] = useState(false);
 
-export const NoteEditor = ({ isPaletteVisible }) => {
-	const [noteColor, setNoteColor] = useState("note__blue");
+	const [alert, setAlert] = useState({
```

Comment:
Try to make a global state of the Alert Component so that you don't manage a state for an alert at every component.

Reply:
Yeah will work on that once all the functionality is done.